app/build.gradle

```diff
@@ -96,7 +96,7 @@ buildscript {
 apply plugin: 'org.greenrobot.greendao'
 
 greendao {
-    schemaVersion 5
+    schemaVersion 9
```

Comment:
Why did this jump so much?

Reply:
Because I were implementing new APIs and new entities step by step, so I had to increase schemaVersion a few times, but now I can change increasing of schemaVersion only per 1.

Reply:
Just keep in mind that users will be upgrading from somewhere between 3-5 to 9, the onupgrade will need to upgrade every single part of the database for between 5-9.

Reply:
No problem, I will fix it)